modules/compiler/utils/include/compiler/utils/replace_wgc_pass.h

```diff
@@ -39,7 +39,13 @@ namespace utils {
 /// target making use of that pass.
 class ReplaceWGCPass final : public llvm::PassInfoMixin<ReplaceWGCPass> {
  public:
+  ReplaceWGCPass(bool scansOnly = false) : scans_only(scansOnly) {}
```

Comment:
This property should be settable on the command line(e.g. `--passes=replace-wgc<scans-only>`) so we can test it in both modes.

Reply:
cool, i was wondering if we could do that

Reply:
you have to add support for it in the pass machinery / .def file